metadata_service/proxy/atlas_proxy.py

```diff
@@ -652,6 +689,53 @@ def _parse_date(self, date: int) -> Optional[int]:
         except Exception:
             return None
 
+    def _get_readers(self, qualified_name: str, top: Optional[int] = None) -> List[Reader]:
```

Comment:
small nit: 
top: Optional[int] = 15? 
so you don't need to do top or 15 in L696

Reply:
fixed